docs/user-guide/pod-security-policy/psp.yaml

```diff
@@ -1,28 +1,18 @@
-{
-  "kind": "PodSecurityPolicy",
-  "apiVersion":"extensions/v1beta1",
-  "metadata": {
-    "name": "permissive"
-  },
-  "spec": {
-      "seLinux": {
-          "rule": "RunAsAny"
-      },
-      "supplementalGroups": {
-          "rule": "RunAsAny"
-      },
-      "runAsUser": {
-          "rule": "RunAsAny"
-      },
-      "fsGroup": {
-          "rule": "RunAsAny"
-      },
-      "HostPorts": [
-          {
-          "min": 8000,
-          "max": 8080
-          }
-      ],
-      "volumes": ["*"]
-  }
-}
+apiVersion: extensions/v1beta1
```

Comment:
hmm, just saw this this morning too https://github.com/kubernetes/kubernetes.github.io/pull/3003  so probably need to make sure we're not stepping on toes between the two